libs/utils/utils.js

```diff
@@ -144,6 +144,27 @@ export const [setConfig, getConfig] = (() => {
         // eslint-disable-next-line no-console
         console.log('Invalid or missing locale:', e);
       }
+      if (Array.isArray(config.contentRoot)) {
```

Comment:
Break out into another small func and make use of config.locale instead - something like:
```
const getContentRoot = (contentRoot, prefix, pathname) => {
  if (Array.isArray(contentRoot)) {
    const rootDir = pathname.split('/')[prefix ? 2 : 1];
    if (contentRoot.includes(rootDir)) {
      return `${prefix}/${rootDir}`;
    } else {
      return prefix;
    }
  }
  return contentRoot;
}

config.contentRoot = getContentRoot(config.contentRoot, config.locale.prefix, pathname);
```